index.js

```diff
@@ -14,11 +14,12 @@
 
 const { AssetCompute } = require("./lib/assetcompute");
 const { AssetComputeEventEmitter } = require("./lib/eventemitter");
-const { createAssetComputeClient, AssetComputeClient } = require("./lib/client");
+const { AssetComputeClient } = require("./lib/client");
+const { YamlConverter } = require('./lib/yamlconverter');
```

Comment:
I think `IntegrationConfiguration` replaces this. It wouldn't be a bad idea to export the integration config anyways